scss/facet-list.scss

```diff
@@ -568,7 +572,7 @@ $facetlist-term-block-height: default !default;
                     display: block;
                 }
 
-                &.selected > a {
+                &.selected > a, &.selected > a .facet-item-group-header {
```

Comment:
Small nitpick here, would be good to put the second set of selectors on the second line, so it's obvious that it's two separate conditions at a glance.